common-lib/src/main/resources/logback.xml

```diff
@@ -2,11 +2,20 @@
 
     <conversionRule conversionWord="coloredLevel" converterClass="play.api.libs.logback.ColoredLevel" />
 
+    <if condition='property("STAGE").contains("DEV")'>
```

Comment:
Haven't seen this before - where does property come from and how is it able to access `STAGE`?

Reply:
It's defined here: https://github.com/guardian/grid/blob/master/.sbtopts#L3